src/lib/select/select.spec.ts

```diff
@@ -19,12 +19,9 @@ describe('MdSelect', () => {
       declarations: [BasicSelect, NgModelSelect, ManySelects, NgIfSelect],
       providers: [
         {provide: OverlayContainer, useFactory: () => {
-          overlayContainerElement = document.createElement('div');
+          overlayContainerElement = document.createElement('div') as HTMLElement;
+          overlayContainerElement.classList.add('cdk-overlay-container');
```

Comment:
Not necessary for this PR: we should make a `FakeOverlayContainer` and use that everywhere